lib/src/user.dart

```diff
@@ -39,6 +39,11 @@ class User {
     return _app ??= AppInternal.create(realmCore.userGetApp(_handle));
   }
 
+  /// Gets an [ApiKeyClient] instance that exposes functionality for managing
+  /// user API keys.
+  /// [API Keys Authentication Docs](https://docs.mongodb.com/realm/authentication/api-key/)
+  late final ApiKeyClient apiKeys = ApiKeyClient._(this);
```

Comment:
I prefer `apiKeys` because it reads nicer - `user.apiKeys.create`. If we name that `apiKeysClient`, the same API will read like we're creating a client, so we'll need to then suffix all the APIs with `ApiKey` - like `createApiKey`, etc.

Reply:
that's a good point. my point was that `apiKeys` reads like an array. `user.apiKeys`.  so one might expect `length` to exists `user.apiKeys.length`.